disco_aws_automation/disco_autoscale.py

```diff
@@ -319,39 +349,41 @@ def _create_new_launchconfig(self, hostclass, launch_config):
             user_data=launch_config.user_data,
             associate_public_ip_address=launch_config.associate_public_ip_address)
 
-    def update_snapshot(self, hostclass, snapshot_id, snapshot_size):
-        '''Updates an existing autoscaling group to use a different snapshot'''
-        launch_config = self.get_launch_config_for_hostclass(hostclass)
+    def update_snapshot(self, snapshot_id, snapshot_size, hostclass=None, group_name=None):
```

Comment:
I didn't really worry too much about snapshotting instances in this case. If we wanted to disable automatic snapshotting of 'test' instances, we'd probably want to do that in `/opt/wgen/bin/take_snapshot.sh` by checking the `is_testing` value.